src/_data/menuItems.json

```diff
@@ -0,0 +1,86 @@
+[
+    {
+        "name": "Schedule",
```

Comment:
Remove this from main nav and add inside Program and add a button in hero section "View Schedule"